src/System.Management.Automation/engine/SessionState.cs

```diff
@@ -305,6 +306,89 @@ internal void InitializeFixedVariables()
                     RunspaceInit.PSHostDescription);
             this.GlobalScope.SetVariable(v.Name, v, asValue: false, force: true, this, CommandOrigin.Internal, fastPath: true);
 
+            // $PSClipboardActions
+            Hashtable clipboardActions = null;
+
+            if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+            {
+                clipboardActions = new Hashtable {
+                    { "Clip", new Hashtable {
+                        { "UseWindowsClipboard", true }
```

Comment:
Not a fan of having to use `Hashtable`s (instead of dicts) because `Clip`/`Paste`s keys can be `strings`, `string[]`s, or bools, and they all have to be manually re-cast in `Clipboard.cs`. Is there a better way to do this, or is a perfect solution going to require a seperate class (see point 2 in https://github.com/PowerShell/PowerShell/pull/24049#issuecomment-2356528249 )?